azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -141,6 +144,23 @@ public void alertOnFailedUpdate() throws Exception {
         .isEqualToIgnoringWhitespace(this.message.getBody());
   }
 
+  @Test
+  public void testAlertOnJobPropertyOverridden() throws Exception {
```

Comment:
Readability: Although we verify the interaction of the addAllToAddress and setSubject methods, the test method name doesn't say about it. It would help if we rename the test method.

Reply:
changed to testOverrideEmailAddressAndSubject